modelon/impact/client/entities/case.py

```diff
@@ -466,6 +466,12 @@ def get_artifact(self, artifact_id: str) -> Tuple[bytes, str]:
 
         return result, file_name
 
+    def get_artifact_ids(self):
```

Comment:
I think it would be even better to even have: `get_artifacts() -> list[CustomArtifact]` to future proof it for even more fields. Usage would look something like:
```
>case.get_artifacts()
[CustomArtifact(id='ABCD', download_as='ABCD.mat')]
>case.get_artifacts()[0].download()  # Can be implemented later if we do not want to this right now
```